composer.json

```diff
@@ -86,10 +86,10 @@
 			"php ./vendor/bin/phpcs -i"
 		],
 		"check-cs": [
-			"php ./vendor/bin/phpcs"
+			"php ./vendor/bin/phpcs -d error_reporting=\"E_ALL&~E_DEPRECATED\""
 		],
 		"fix-cs": [
-			"php ./vendor/bin/phpcbf"
+			"php ./vendor/bin/phpcbf -d error_reporting=\"E_ALL&~E_DEPRECATED\""
```

Comment:
I have PHP 8.2 installed locally and was getting a ton of error such as:

```
FILE: /Users/blakewilson/wpengine/wp-graphql/src/Type/ObjectType/User.php
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | An error occurred during processing; checking has been aborted. The error message was: trim(): Passing null to parameter #1 ($string) of type string is deprecated in
   |       | /Users/blakewilson/wpengine/wp-graphql/vendor/wp-coding-standards/wpcs/WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php on line 280 (Internal.Exception)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 266ms; Memory: 22MB

Script php ./vendor/bin/phpcs handling the check-cs event returned with error code 1
```

when running `fix-cs`/`check-cs`. I came across this GitHub issue that states that PHP_CodeSniffer is currently unsupported in 8.0 and above, unless you are using their `develop` branch.

By adding these following flags, they ignore these deprecated errors as recommended in the issue:

https://github.com/WordPress/WordPress-Coding-Standards/issues/2035

Let me know if this is something worth adding. If not I can remove it, or open a new PR for it.

Reply:
@blakewilson will that only silence issues _with_ PHPCS, or also errors caused WPGraphQL's codebase? 

We're still throwing  notices in 8.1+ so if this change will obsfucate them, it might be better we recommend a local [`phpcs.xml`](https://github.com/WordPress/WordPress-Coding-Standards/issues/2035#issuecomment-1325532520) (so ya don't need to keep changing composer.json) until we're forward-compatible.

Reply:
@justlevine I like your idea of using `phpcs.xml`, I think that's a more sustainable avenue here. Going to rollback this change.